docs/supported-libraries.md

```diff
@@ -17,111 +17,134 @@ or [contributing](../CONTRIBUTING.md).
+| [JMS](https://javaee.github.io/javaee-spec/javadocs/javax/jms/package-summary.html)                                                         | 1.1+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     |                                                                                        |
+| [JSP](https://javaee.github.io/javaee-spec/javadocs/javax/servlet/jsp/package-summary.html)                                                 | 2.3+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     |                                                                                        |
+| [Kotlin Coroutines](https://kotlinlang.org/docs/coroutines-overview.html)                                                                   | 1.0+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     |                                                                                        |
+| [Ktor](https://github.com/ktorio/ktor)                                                                                                      | 1.0+ and 2.0+                 | [opentelemetry-ktor-1.0](../instrumentation/ktor/ktor-1.0/library),<br>[opentelemetry-ktor-2.0](../instrumentation/ktor/ktor-2.0/library)                                                                                                                                                                                                                                               |                                                                                        |
```

Comment:
I don't think it's needed to call out 2.0 separately for auto-instrumented versioning column
```suggestion
| [Ktor](https://github.com/ktorio/ktor)                                                                                                      | 1.0+                 | [opentelemetry-ktor-1.0](../instrumentation/ktor/ktor-1.0/library),<br>[opentelemetry-ktor-2.0](../instrumentation/ktor/ktor-2.0/library)                                                                                                                                                                                                                                               |                                                                                        |
```